crates/hyperqueue/src/client/commands/journal/mod.rs

```diff
@@ -60,20 +60,27 @@ struct ExportOpts {
 pub async fn command_journal(gsettings: &GlobalSettings, opts: JournalOpts) -> anyhow::Result<()> {
     match opts.command {
         JournalCommand::Export(opts) => export_json(opts),
-        JournalCommand::Replay => stream_json(gsettings, false).await,
-        JournalCommand::Stream => stream_json(gsettings, true).await,
+        JournalCommand::Replay => stream_json(gsettings, true, false).await,
+        JournalCommand::Stream => stream_json(gsettings, true, true).await,
```

Comment:
What happens if you select live events false and past events false? This would be better as an enum maybe?